math-dojo/single-page-web-app
# add functionality to render equations in question page

## Purpose

* Adds the functionality to render equations in the UI
* Renders equations stored in questionBody attribute of question object in question page UI
* Minor backend compatibility name changes



## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
npm start
go to link http://localhost:4200/questions/try-me-first
check that equation renders properly
